pages/examples/content/fr/style-overrides.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Accepter les surcharges de style
+subtitle: Comment exposer une API pour surcharger les styles d'un composant
+contributors:
+  - kentcdodds
+  - igitscor
+---
+
+Voici une explication de comment, avec un composant réutilisble, exposer
+un mécanisme pour surcharger le style d'autres composants à l'aide d'une
+propriété `styleOverrides`.
+
+La partie essentielle est de passer `styleOverrides` à la propriété `theme` du
+`ThemeProvider` glamorous. Parce que le `theme` sera utilisé pour d'autres éléments
+de votre application, il est toujours bon de rajouter un namespace
```

Comment:
`namespace` should be understandable by most french developers, as it is a keyword in many languages (C++, php...), but "Espace de nom" is probably the right translation.